listings/applications/simple_storage_starknetjs/index.js

```diff
@@ -0,0 +1,30 @@
+import { Account, RpcProvider, json, Contract } from 'starknet';
+import fs from 'fs';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+const provider = new RpcProvider({ nodeUrl: 'https://starknet-sepolia.public.blastapi.io' });
```

Comment:
Could you use nethermind rpc provider: `https://free-rpc.nethermind.io/sepolia-juno`